specs/latest/2.0/index.html

```diff
@@ -3308,6 +3310,22 @@ <h2 class="no-toc">Table of contents</h2>
         OpenGL ES 3.0.4 &sect;3.9.2</a>).
     </div>
 
+    <h3><a name="FRAMEBUFFER_COLOR_ATTACHMENTS">Framebuffer color attachments</a></h3>
+
+    <p>
+        If a color image is attached to more than one attachment point in a framebuffer,
```

Comment:
nit: two more instances of "color image"->"image" and "attachment point"->"color attachment point" in this paragraph.